### PR TITLE
Replace is_a() with instanceof operator

### PR DESCRIPTION
**Pull request by mundschenk-at**
Replaces uses of `is_a()` function with with the faster (and more readable) `instanceof` operator.

### CODE REVIEW

File: php/utils-wp.php [hunk 1]
```diff
@@ -220,14 +220,14 @@ function wp_get_cache_type() {
 
 	if ( ! empty( $_wp_using_ext_object_cache ) ) {
 		// Test for Memcached PECL extension memcached object cache (https://github.com/tollmanz/wordpress-memcached-backend)
-		if ( isset( $wp_object_cache->m ) && is_a( $wp_object_cache->m, 'Memcached' ) ) {
+		if ( isset( $wp_object_cache->m ) && ( $wp_object_cache->m instanceof \Memcached ) ) {
```
Comment (schlessera) [inline php/utils-wp.php]: Parentheses are not needed here:
```suggestion
		if ( isset( $wp_object_cache->m ) && $wp_object_cache->m instanceof \Memcached ) {
```

File: php/utils-wp.php [hunk 2]
```diff
@@ -220,14 +220,14 @@ function wp_get_cache_type() {
...
 			$message = 'Memcached';
 
 			// Test for Memcache PECL extension memcached object cache (http://wordpress.org/extend/plugins/memcached/)
 		} elseif ( isset( $wp_object_cache->mc ) ) {
 			$is_memcache = true;
 			foreach ( $wp_object_cache->mc as $bucket ) {
-				if ( ! is_a( $bucket, 'Memcache' ) && ! is_a( $bucket, 'Memcached' ) ) {
+				if ( ( ! $bucket instanceof \Memcache ) && ( ! $bucket instanceof \Memcached ) ) {
```
Comment (schlessera) [inline php/utils-wp.php]: Parentheses are not needed here:
```suggestion
				if ( ! $bucket instanceof \Memcache && ! $bucket instanceof \Memcached ) {
```

File: php/utils-wp.php [hunk 3]
```diff
@@ -249,11 +249,11 @@ function wp_get_cache_type() {
 			$message = 'APC';
 
 			// Test for Redis Object Cache (https://github.com/alleyinteractive/wp-redis)
-		} elseif ( isset( $wp_object_cache->redis ) && is_a( $wp_object_cache->redis, 'Redis' ) ) {
+		} elseif ( isset( $wp_object_cache->redis ) && ( $wp_object_cache->redis instanceof \Redis ) ) {
```
Comment (schlessera) [inline php/utils-wp.php]: Parentheses are not needed here:
```suggestion
		} elseif ( isset( $wp_object_cache->redis ) && $wp_object_cache->redis instanceof \Redis ) {
```

File: php/utils-wp.php [hunk 4]
```diff
@@ -249,11 +249,11 @@ function wp_get_cache_type() {
...
 			$message = 'Redis';
 
 			// Test for WP LCache Object cache (https://github.com/lcache/wp-lcache)
-		} elseif ( isset( $wp_object_cache->lcache ) && is_a( $wp_object_cache->lcache, '\LCache\Integrated' ) ) {
+		} elseif ( isset( $wp_object_cache->lcache ) && ( $wp_object_cache->lcache instanceof \LCache\Integrated ) ) {
```
Comment (schlessera) [inline php/utils-wp.php]: Parentheses are not needed here:
```suggestion
		} elseif ( isset( $wp_object_cache->lcache ) && $wp_object_cache->lcache instanceof \LCache\Integrated ) {
```

File: php/utils-wp.php (hunk 2 above)
Comment (mundschenk-at) [reply php/utils-wp.php]: I'll change that of course, but here I think the expression is more readable with the parentheses.

**Comment by mundschenk-at** [comment]
The parentheses have all been removed.

**Comment by schlessera** [comment]
Thanks for the refactor, @mundschenk-at !

**Pull request merged**